src/types/capsule.rs

```diff
@@ -123,8 +123,9 @@ impl PyCapsule {
 
     /// Imports an existing capsule.
     ///
-    /// The `name` should match the path to the module attribute exactly in the form
-    /// of `"module.attribute"`, which should be the same as the name within the capsule.
+    /// If this capsule represents a module attribute, the `name` should match the path
+    /// to the module attribute exactly in the form of `"module.attribute"`, which should
+    /// be the same as the name within the capsule.
```

Comment:
I felt this paragraph was oddly prescriptive, given there are other use cases for `PyCapsules` than module attributes. (For example, we use them in Arrow to transport Array in [the Arrow PyCapsule Interface](https://arrow.apache.org/docs/format/CDataInterface/PyCapsuleInterface.html).) If you feel otherwise, I can revert this change.

Reply:
It comes from the documentation for [`PyCapsule_Import`](https://docs.python.org/3/c-api/capsule.html#c.PyCapsule_Import)

Based on your adjustment, you wanted to clarify that capsules can be used for things other than module attributes. This is true, though this function only succeeds for module attributes as far as I am aware?

Reply:
Ah I see what you are saying. I was looking at this because it's the only API where you can pass a name and have that validated. So instead, I think we want to call `GetPointer`, right?